CBLClient/Apps/CBLTestServer-Android/app/src/main/java/com/couchbase/CouchbaseLiteServ/server/RequestHandler/ReplicatorConfigurationRequestHandler.java

```diff
@@ -0,0 +1,260 @@
+        }
+        System.out.println(args);
+        if (pinnedservercert != null){
+            Context context = com.couchbase.CouchbaseLiteServ.MainActivity.getAppContext();
+            byte[] ServerCert = this.getPinnedCertFile(context);
+            // Set pinned certificate.
+            config.setPinnedServerCertificate(ServerCert);
+        }
+        return config;
+    }
+
+    public ReplicatorConfiguration create(Args args) {
+        ReplicatorConfiguration config = args.get("configuration");
+        return config;
+    }
+
+    public Authenticator getAuthenticator(Args args){
+        ReplicatorConfiguration replicatorConfiguration = args.get("configuration");
+        return replicatorConfiguration.getAuthenticator();
+    }
+
+    public List<String>  getChannels(Args args){
+        ReplicatorConfiguration replicatorConfiguration = args.get("configuration");
+        return replicatorConfiguration.getChannels();
+    }
+
+    /*public ConflictResolver getConflictResolver(Args args){
+        ReplicatorConfiguration replicatorConfiguration = args.get("configuration");
+        return replicatorConfiguration.getConflictResolver();
+    }*/
+
+    public Database getDatabase(Args args){
+        ReplicatorConfiguration replicatorConfiguration = args.get("configuration");
+        return replicatorConfiguration.getDatabase();
+    }
+
+    public List<String> getDocumentIDs(Args args){
+        ReplicatorConfiguration replicatorConfiguration = args.get("configuration");
+        return replicatorConfiguration.getDocumentIDs();
+    }
+
+    public byte[] getPinnedServerCertificate(Args args){
+        ReplicatorConfiguration replicatorConfiguration = args.get("configuration");
+        return replicatorConfiguration.getPinnedServerCertificate();
+    }
+
+    public String getReplicatorType(Args args){
+        ReplicatorConfiguration replicatorConfiguration = args.get("configuration");
+        return replicatorConfiguration.getReplicatorType().toString();
+    }
+
+    public String getTarget(Args args){
+        ReplicatorConfiguration replicatorConfiguration = args.get("configuration");
+        return replicatorConfiguration.getTarget().toString();
+    }
+
+    public Boolean isContinuous(Args args){
+        ReplicatorConfiguration replicatorConfiguration = args.get("configuration");
+        return replicatorConfiguration.isContinuous();
+    }
+
+    public void setAuthenticator(Args args){
+        ReplicatorConfiguration replicatorConfiguration = args.get("configuration");
+        Authenticator authenticator = args.get("authenticator");
+        replicatorConfiguration.setAuthenticator(authenticator);
+    }
+
+    public void setChannels(Args args){
+        ReplicatorConfiguration replicatorConfiguration = args.get("configuration");
+        List<String> channels = args.get("channels");
+        replicatorConfiguration.setChannels(channels);
+    }
+
+    /*public void setConflictResolver(Args args){
```

Comment:
Can this commented code be removed?

Reply:
will wait for 6 months and remove, if dev get custom conflict resolver later , this might be needed